client/my-sites/upgrades/domain-management/email/index.jsx

```diff
@@ -41,8 +43,14 @@ const Email = React.createClass( {
 	},
 
 	render() {
+		const showPlanFeatures = config.isEnabled( 'manage/plan-features' );
+
+		const classes = classNames( 'domain-management-email', {
+			'is-wide-layout': showPlanFeatures
```

Comment:
Let's add a `wide` or `wideLayout` prop to main, that adds an `is-wide-layout` class. PlanActions can take in a prop for the `has-personal-plans` class.